### ISSUE TITLE
Convert EfficientDet  to onnx

### ISSUE THREAD

**Issue by Yuye-Castle** [neverrop/Deep-learning#1]

**Comment by Yuye-Castle** [comment]
`作者在utils_extra.py 里自己定义的两个类（staticsamepadding和pooling）里面的forward函数里包含了python，numpy的代码，并且是根据x动态确定shape的，这些是不能直接导出成onnx的。解决方法是把image_shape 当作参数传入init里。比如：
class Conv2dStaticSamePadding(nn.Module):
    """
    created by Zylo117
    The real keras/tensorflow conv2d with same padding
    """

    def __init__(self, in_channels, out_channels, kernel_size, stride=1, bias=True, groups=1, dilation=1, image_size=None, **kwargs):
        super().__init__()
        self.conv = nn.Conv2d(in_channels, out_channels, kernel_size, stride=stride,
                              bias=bias, groups=groups)
        self.stride = self.conv.stride
        self.kernel_size = self.conv.kernel_size
        self.dilation = self.conv.dilation

        if isinstance(self.stride, int):
            self.stride = [self.stride] * 2
        elif len(self.stride) == 1:
            self.stride = [self.stride[0]] * 2

        if isinstance(self.kernel_size, int):
            self.kernel_size = [self.kernel_size] * 2
        elif len(self.kernel_size) == 1:
            self.kernel_size = [self.kernel_size[0]] * 2

        assert image_size is not None
        h, w = image_size if type(image_size) == list else [image_size, image_size]

        h_step = math.ceil(w / self.stride[1])
        v_step = math.ceil(h / self.stride[0])
        h_cover_len = self.stride[1] * (h_step - 1) + 1 + (self.kernel_size[1] - 1)
        v_cover_len = self.stride[0] * (v_step - 1) + 1 + (self.kernel_size[0] - 1)

        extra_h = h_cover_len - w
        extra_v = v_cover_len - h

        left = extra_h // 2
        right = extra_h - left
        top = extra_v // 2
        bottom = extra_v - top

        self.static_padding = nn.ZeroPad2d((left, right, top, bottom))

    def forward(self, x):
        x = self.static_padding(x)
        x = self.conv(x)
        
        return x`

- How to set input_size of Conv2dStaticSamePadding constructor
- how to modify Anchor  class
Can you modify your modified class?
